banners/templatetags/banners.py

```diff
@@ -0,0 +1,23 @@
+from django import template
+from django.template.loader import render_to_string
+
+from banners.models import Banner
+
+register = template.Library()
+
+
+@register.simple_tag
+def render_active_banner(psf_pages_only=True):
+    if not psf_pages_only:
```

Comment:
If I understood this correctly, the following line would cover both if and else statements:

```py
banner = Banner.objects.filter(active=True, psf_pages_only=psf_pages_only).first()
```

Reply:
little trickier than that, since we accept either value for the PSF pages. so filtering on psf_pages_only makes general banners _not_ show.